### PR TITLE
Doc: Response

### PR DESCRIPTION
**Pull request by ShrutiVerma97**

### CODE REVIEW

File: docs/website/docs/v1.x/dsl/response/index.md [hunk 1]
```diff
@@ -1 +1,71 @@
...
+# Response
+
+**ZIO HTTP** `Response` is designed to encode HTTP Response.
+It supports all HTTP status codes and headers along with custom methods and headers (as defined in [RFC2616](https://datatracker.ietf.org/doc/html/rfc2616) )
+
+## Creating a Response
+
+`Response` can be created with `status`, `headers` and `data`.  
+
+The below snippet creates a response with default params, `status` as `Status.OK`, `headers` as `Headers.empty`, `data` as `HttpData.Empty`.
```
Comment (tusharmath) [inline docs/website/docs/v1.x/dsl/response/index.md]:
```suggestion
The below snippet creates a response with default params, `status` as `Status.OK`, `headers` as `Headers.empty`, and `data` as `HttpData.Empty`.
```

File: docs/website/docs/v1.x/dsl/response/index.md [hunk 2]
```diff
@@ -1 +1,71 @@
...
+```scala
+ val request: Request = Request()
```
Comment (tusharmath) [inline docs/website/docs/v1.x/dsl/response/index.md]: Request example in Response?

File: docs/website/docs/v1.x/dsl/response/index.md [hunk 3]
```diff
@@ -1 +1,71 @@
...
+```scala
+ val res: Response = Response.ok
+```
+
+- `status` creates an empty response with provided status code.
+```scala
+ val res: Response = Response.status(Status.CONTINUE)
+```
+
+### Response with Content-Type
```
Comment (tusharmath) [inline docs/website/docs/v1.x/dsl/response/index.md]:
```suggestion
### Specialized Response Constructors
```

File: docs/website/docs/v1.x/dsl/response/index.md [hunk 4]
```diff
@@ -1 +1,71 @@
...
+ val res: Response = Response.text("hey")
+```
+- `json` creates a response with content-type set to application/json
+```scala
+ val res: Response = Response.json("""{"greetings": "Hello World!"}""")
+```
+- `html` creates a response with content-type set to text/html
+```scala
+ val res: Response = Response.html(Html.fromString("html text"))
+```
```
Comment (tusharmath) [inline docs/website/docs/v1.x/dsl/response/index.md]: We can talk about the headers and status code here.

File: docs/website/docs/v1.x/dsl/response/index.md [hunk 5]
```diff
@@ -1 +1,71 @@
...
+```scala
+ val res: Response = Response.fromHttpError(HttpError.BadRequest())
+```
+
+### Response from Socket
+
+`fromSocket` creates an effectful response from `Socket` provided
+```scala
+ val socket: Socket[Any, Nothing, Any, WebSocketFrame] = Socket.succeed(WebSocketFrame.text("Greetings!"))
+ val res: ZIO[Any, Nothing, Response] = Response.fromSocket(socket)
```
Comment (tusharmath) [inline docs/website/docs/v1.x/dsl/response/index.md]: I think we should drop this method, constructors should not create a ZIO.

File: docs/website/docs/v1.x/dsl/response/index.md [hunk 6]
```diff
@@ -1 +1,71 @@
...
+```
+
+### Response from SocketApp
```
Comment (tusharmath) [inline docs/website/docs/v1.x/dsl/response/index.md]: Can be part of the same section — From Socket

File: docs/website/docs/v1.x/dsl/response/index.md [hunk 7]
```diff
@@ -1 +1,71 @@
...
+ val socketApp = SocketApp(socket)
+ val res: ZIO[Any, Nothing, Response] = Response.fromSocketApp(socketApp)
+```
+
+## Adding Cookie to Response
+
+`addCookie` adds cookies in the headers of the response.
+```scala
+ val cookie = Cookie("key", "value").withMaxAge(5 days)
+ val res = Response.ok.addCookie(cookie.withPath(!! / "cookie").withHttpOnly)
```
Comment (tusharmath) [inline docs/website/docs/v1.x/dsl/response/index.md]: Keep the example for `Cookie` creation simple. Idea is to make someone understand how to add `Cookie` and not how to create a Cookie.

File: docs/website/docs/v1.x/dsl/response.md [hunk 8]
```diff
@@ -1,3 +1,67 @@
...
+```scala
+ val res: Response = Response.html(Html.fromString("html text"))
+```
+
+### Specialized Response Operators
+
+- `setStatus` to update the `status` of `Response`
+
+```scala
+val res: Response = Response.ok.setStatus(Status.NOT_FOUND)
```
Comment (amitksingh1490) [inline docs/website/docs/v1.x/dsl/response.md]: use setstatus on sume other constructor example `text`

**Review by amitksingh1490 — APPROVED** [review]

**Pull request merged**